sound/soc/codecs/max98373-sdw.h

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: GPL-2.0
+ *  Copyright (c) 2020 Maxim Integrated
+ */
+
+#ifndef _MAX98373_SDW_H
+#define _MAX98373_SDW_H
+
+#include "max98373.h"
+
+struct sdw_stream_data {
```

Comment:
I really don't think you want a struct with a name like that defined in a codec header

Reply:
OK. Removed from header file.